lib/assert.js

```diff
@@ -151,6 +153,29 @@ x.ifError = function (err, msg) {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
 
+x.snapshot = function (tree, optionalMessage, match, snapshotStateGetter) {
+	// set defaults - this allows tests to mock deps easily
+	var toMatchSnapshot = match || jestSnapshot.toMatchSnapshot;
+	var getState = snapshotStateGetter || snapshotState.get;
```

Comment:
@lithin this needs to be `snapshotState.get.bind(snapshotState)` or `snapshotState.get.bind(this)` as it uses `this` internally and I keep getting the error: `Error: Cannot read property 'state' of undefined`.